net-dns/dnscap/dnscap-20130814.ebuild

```diff
@@ -1,12 +1,10 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-
-inherit eutils
+EAPI=7
 
 DESCRIPTION="dnscap is a network capture utility designed specifically for DNS traffic"
```

Comment:
if you gonna change the package itself please also set proper description, it should not contain the package name

Reply:
Updated the Description too